test/integration/services/lnd/utils.spec.ts

```diff
@@ -80,16 +83,37 @@ describe("lndUtils", () => {
       }
     })
 
-    baseLogger.debug((await getForwards({ lnd: lnd1 })).forwards, "forwards")
-
     const date = Date.now() + 60 * 60 * 1000 * 24 * 2
     jest.spyOn(global.Date, "now").mockImplementation(() => new Date(date).valueOf())
 
+    const startDate = new Date(0)
+    startDate.setUTCHours(0, 0, 0, 0)
+
+    const endDate = new Date(Date.now() - 864e5)
+    endDate.setUTCHours(0, 0, 0, 0)
+
+    const after = startDate.toISOString()
+    const before = endDate.toISOString()
+
+    const totalFees = (await getForwards({ lnd: lnd1, after, before })).forwards
+      .map((f) => Number(f.fee_mtokens))
+      .reduce((acc, val) => acc + val)
```

Comment:
Minor nitpick but I'd generally avoid multiple scanning when possible: 

`.reduce((acc, val) => acc + Number(val.fee_mtokens))`

Reply:
It is not possible do it in that way, in that way acc would be forward type so it crash when you try to sum the fee

Reply:
I'll check if setting initial value fixes that, but I tried that way and it didn't work

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/411/commits/27007f331fb86f9c96147622aa2aefcbfdfe8686 with initial value. `.reduce((acc, val) => acc + Number(val.fee_mtokens), 0)`